digiwf-engine/digiwf-engine-service/src/main/resources/bausteine/dms/dokumentstornieren/DokumentStornierenV01.bpmn

```diff
@@ -67,8 +67,8 @@ Hinweis: Diese E-Mail wurde maschinell erstellt. Bitte antworten Sie nicht auf d
           </camunda:inputParameter>
         </camunda:inputOutput>
         <camunda:taskListener event="complete">
-          <camunda:script scriptFormat="javascript">sachbearbeitung= task.execution.getVariable("FormField_Neue_Sachbearbeitung");
-task.execution.setVariable("dms_user_login",sachbearbeitung);</camunda:script>
+          <camunda:script scriptFormat="javascript">sachbearbeitung= task.getVariable("FormField_Neue_Sachbearbeitung");
+task.setVariable("dms_user_login",sachbearbeitung);</camunda:script>
```

Comment:
Shouldn't it be "task.**getExecution(**).getVariable(...)"? Otherwise the behavior changes because the scope is different.